wikidp/routes/pages.py

```diff
@@ -70,7 +72,8 @@ def route_item_contribute(qid):
     """Handles a user's contributed statements."""
     selected_item, options, schemas = get_item_context(qid, with_claims=False)
     if selected_item:
-        return render_template('item_contribute.html', item=selected_item, options=options, schemas=schemas,
+        return render_template('item_contribute.html', item=selected_item,
```

Comment:
I don't think I updated my text-editor's line length from another project (at 120); for future references, what line length should we be using? (perhaps 80?)